redpipe/connections.py

```diff
@@ -54,15 +55,31 @@ def get(cls, name: Optional[str] = None) -> redis.client.Pipeline:
         except KeyError:
             raise InvalidPipeline('%s is not configured' % name)
 
+    @classmethod
+    def get_client(cls, name: Optional[str]) -> redis.Redis:
+        """
+        Returns a low-level Redis client for the connection
+        Called by the redpipe.scripts module.
+        Don't call this directly.
+
+        :param name: str
+        :return: redis.Redis (or subclass, e.g. RedisCluster)
+        """
+        try:
+            return cls.clients[name]
+        except KeyError:
+            raise InvalidPipeline('%s is not configured' % name)
+
     @classmethod
     def connect(cls, pipeline_method: Callable,
-                name: Optional[str] = None) -> None:
+                name: Optional[str] = None, client=None) -> None:
```

Comment:
```suggestion
                name: Optional[str] = None, client: Optional[redis.Redis]=None) -> None:
```

Reply:
The problem is that the client could be e.g. a `redis.RedisCluster`, which doesn't directly inherit from `redis.Redis`.  But there are a lot of interface-style superclasses, so let me see if I can find a common ancestor.

Reply:
Oh hmm but the dictionary is typed with `redis.Redis`...I wonder if that's a bug though 😅

Reply:
humm I was checking it too, it is also typed as `Dict[Optional[str]...` which is a little weird

Reply:
ooh, that feels so wrong, but works 🤯 

<img width="249" alt="image" src="https://github.com/user-attachments/assets/887ff0b4-76e9-4c49-9c10-922923891d79">